readme.md

```diff
@@ -2,6 +2,16 @@
 
 goflyway is a tunnel proxy helping you fly across the wall. It is based entirely on HTTP protocol without any other 3rd party libraries. 
 
+## Install
+
+`go get` will show error " local import in non-local package", instead you should clone into directory outsite your `$GOPATH` directory:
+
+```
+$ git clone https://github.com/coyove/goflyway
+$ cd goflyway
+$ go install
```

Comment:
Since we git cloned the source, `go install` won't work outside GOPATH, `go build main.go` would be fine

Reply:
Even in GOPATH it is complaining on local imports!! -_-, 

```
main.go:4:2: local import "./config" in non-local package
main.go:5:2: local import "./logg" in non-local package
main.go:6:2: local import "./lookup" in non-local package
main.go:7:2: local import "./lru" in non-local package
main.go:8:2: local import "./proxy" in non-local package
```
`go build main.go && mv main $GOPATH/bin/goflyway`  did work